modules/bridge/src/main/ts/ephox/bridge/components/menu/FancyMenuItem.ts

```diff
@@ -4,13 +4,15 @@ import { Result, Fun } from '@ephox/katamari';
 export interface FancyMenuItemSpec {
   type: 'fancymenuitem';
   fancytype: string;
+  initData?: Record<string, any>;
   onAction: (data: any) => void;
 }
 
 export interface FancyMenuItem {
   type: 'fancymenuitem';
   fancytype: keyof FancyActionArgsMap;
   onAction: <K extends keyof FancyActionArgsMap>(data: FancyActionArgsMap[K]) => void;
+  initData?: Record<string, any>;
```

Comment:
No optional items in a final spec please. This should default to an empty object instead of `undefined` basically.
```suggestion
  initData: Record<string, any>;
```